disposable_email_blacklist.conf

```diff
@@ -2076,6 +2135,9 @@ spamfree24.info
 spamfree24.net
 spamfree24.org
 spamgoes.in
+spamgourmet.com
```

Comment:
`spamgourment.*` is in the `whitelist.conf` as it requires a registration before the service can be accessible

Reply:
But it allow to create any number of disposable emails.

Reply:
Done